package.json

```diff
@@ -3,6 +3,9 @@
     "version": "0.12.0-pre",
     "dependencies": {
         "@keep-network/tbtc.js": ">0.12.0-pre <0.12.0-rc",
+        "@ledgerhq/hw-app-eth": "^5.11.0",
+        "@ledgerhq/hw-transport-u2f": "^5.11.0",
```

Comment:
It looks like `web3-subprovider` already [transitively pulls the above two dependencies in](https://github.com/LedgerHQ/ledgerjs/blob/fc435a9611e9e3554d0d4be2939e6da44ba20735/packages/web3-subprovider/package.json#L32-L34); might be worth dropping them from here unless and until we need to do something special with them?